debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4

```diff
@@ -613,6 +613,7 @@ QUERY:                               'QUERY';
 QUICK:                               'QUICK';
 REBUILD:                             'REBUILD';
 RECOVER:                             'RECOVER';
+RECURSIVE:                           'RECURSIVE';
```

Comment:
@ani-sha As you are adding a new keyword, can t be used as table/column name? If yes then this should be reflected in the grammart too - `keywordCanBeId`